src/pages/components/select/style.mdx

```diff
@@ -12,15 +12,15 @@ Inputs come in two different colors. The default input color is `$field-01` and
 is used on `$ui-background` and `$ui-02` page backgrounds. The `--light` version
 input color is `$field-02` and is used on `$ui-01` page backgrounds.
 
-| Class                 | Property   | Color toke  |
-| --------------------- | ---------- | ----------- |
-| `.bx--select-input`   | background | `$field-01` |
-| `.bx--select--light`  | background | `$field-02` |
-| `.bx--select--inline` | background | transparent |
-| `.bx--label`          | text color | `$text-01`  |
-| `.bx--select-input`   | text color | `$text-01`  |
-| `.bx--select--inline` | text color | `$icon-01`  |
-| `.bx--select__arrow`  | fill       | `$icon-01`  |
+| Class                 | Property   | Color token                 |
+| --------------------- | ---------- | --------------------------- |
+| `.bx--select-input`   | background | `$background`               |
+| `.bx--select--light`  | background | `$background: inline theme` |
```

Comment:
And you can delete this row. We're not showing the `light` variants anymore. You'd only us "inline theme" if the components require that version for the default style, like within Modal.